tests/unit/compat/mock.py

```diff
@@ -69,8 +69,7 @@ def _iterate_read_data(read_data):
             # newline that our naive format() added
             data_as_list[-1] = data_as_list[-1][:-1]
 
-        for line in data_as_list:
-            yield line
+        yield from data_as_list
```

Comment:
These can be part of separate PRs, but, none of the compat files are required, since we stopped testing against Py 2.7. This is how it was handled in cisco.nxos: https://github.com/ansible-collections/cisco.nxos/pull/835